### PR TITLE
added statuscake alerts

### PR DESCRIPTION
**Pull request by temitope777**
## Context
added statuscake to prod and staging

## Changes proposed in this pull request

to add status cake alerts 

## Guidance to review
statuscake resources are created for prod and staging

## Link to Trello card

https://trello.com/c/j0ZXrApw/1726-ittms-statuscake-monitoring

### CODE REVIEW

File: terraform/application/statuscake.tf [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module "statuscake" {
+  for_each = var.statuscake_alerts
```
Comment (saliceti) [inline terraform/application/statuscake.tf:2]: We don't need a for_each if there is only 1 entry in the map

File: terraform/application/config/staging.tfvars.json [hunk 2]
```diff
@@ -1,4 +1,13 @@
 {
     "cluster": "test",
-    "namespace": "bat-staging"
+    "namespace": "bat-staging",
+    "statuscake_password_name": "SC-PASSWORD",
+    "infra_key_vault_name": "s189t01-ittms-stg-inf-kv",
+    "key_vault_resource_group":  "s189t01-ittms-stg-rg",
+    "statuscake_alerts": {
+      "alert": {
+        "website_url": [ "https://staging.claim-funding-for-mentor-training.education.gov.uk/healthcheck.json" ],
```
Comment (saliceti) [inline terraform/application/config/staging.tfvars.json:9]: Can we monitor the ingress URL as well?
Such as: https://github.com/DFE-Digital/teacher-services-cloud/blob/main/templates/new_service/terraform/application/statuscake.tf#L6

File: terraform/application/statuscake.tf (hunk 1 above)
Comment (temitope777) [reply terraform/application/statuscake.tf:2]: done

File: terraform/application/config/staging.tfvars.json (hunk 2 above)
Comment (temitope777) [reply terraform/application/config/staging.tfvars.json:9]: done

**Review by saliceti — APPROVED** [review]

**Pull request merged**